test/remove-dependencies.js

```diff
@@ -43,10 +43,10 @@ test('async - multiple types', async t => {
 
 	t.deepEqual(packageJson, omit(fixture, ['dependencies']));
 
-	await removePackageDependencies(temporary, {devDependencies: ['bar']});
+	await removePackageDependencies(temporary, {devDependencies: ['bar', 'baz'], peerDependencies: ['foobar']});
 	packageJson = await readPackage({cwd: temporary, normalize: false});
 
-	t.deepEqual(packageJson, omit(fixture, ['dependencies', 'devDependencies.bar']));
+	t.deepEqual(packageJson, pick(fixture, ['name']));
```

Comment:
Feels simpler to just do this:

```suggestion
	t.deepEqual(packageJson, {name: fixture.name}));
```